### PR TITLE
OCD-4643: start tracking SED interactions in new analytics package

### PR DESCRIPTION
**Pull request by andlar**

### CODE REVIEW

**Review by mattystank — APPROVED** [review]

File: src/app/components/listing/details/sed/sed-task-participants-view.jsx [hunk 1]
```diff
@@ -41,6 +42,10 @@ function ChplSedTaskParticipantsView(props) {
   const classes = useStyles();
 
   const handleClickOpen = () => {
+    eventTrack({
+      event: 'View SED Participant Details',
```
Comment (kekey1) [inline src/app/components/listing/details/sed/sed-task-participants-view.jsx:46]: I don't see this one in the spreadsheet. There is an "Open Participant Details"

File: src/app/components/listing/details/sed/sed-task-view.jsx [hunk 2]
```diff
@@ -112,6 +113,13 @@ function ChplSedTaskView({ task: initialTask }) {
   }, [initialTask]);
 
   const handleAccordionChange = () => {
+    if (!expanded) {
+      eventTrack({
+        event: 'Open SED Task',
```
Comment (kekey1) [inline src/app/components/listing/details/sed/sed-task-view.jsx:118]: Is this "Open SED Testing Task" from the spreadsheet?

**Review by kekey1 — APPROVED** [review]

File: src/app/components/listing/details/sed/sed-task-view.jsx (hunk 2 above)
Comment (mattystank) [reply src/app/components/listing/details/sed/sed-task-view.jsx:118]: Correct, there the same.

File: src/app/components/listing/details/sed/sed-task-participants-view.jsx (hunk 1 above)
Comment (mattystank) [reply src/app/components/listing/details/sed/sed-task-participants-view.jsx:46]: View is correct, I will update the spreadsheet in follow our historical events

File: src/app/components/listing/details/sed/sed-task-view.jsx (hunk 2 above)
Comment (mattystank) [reply src/app/components/listing/details/sed/sed-task-view.jsx:118]: Think of changing this to "View" ? to follow previous pattern

**Review by tmy1313 — APPROVED** [review]

File: src/app/components/listing/details/sed/sed-task-participants-view.jsx (hunk 1 above)
Comment (kekey1) [reply src/app/components/listing/details/sed/sed-task-participants-view.jsx:46]: I updated the spreadsheet

File: src/app/components/listing/details/sed/sed-task-view.jsx (hunk 2 above)
Comment (kekey1) [reply src/app/components/listing/details/sed/sed-task-view.jsx:118]: Updated the spreadsheet to say "View SED Testing Task" but this code needs to be updated to match that.

**Review by kekey1 — CHANGES_REQUESTED** [review]
Marking as request changes so that the spreadsheet events and the events here are made to match.

File: src/app/components/listing/details/sed/sed-task-view.jsx (hunk 2 above)
Comment (andlar) [reply src/app/components/listing/details/sed/sed-task-view.jsx:118]: The spreadsheet is not relevant to this ticket. The spreadsheet is the "truth and final" and will be going through so many revisions before it even gets started. This ticket is to do something like an alpha-version of SED tracking, and nothing in this is going to be live once we actually have a comprehensive suite of analytics

**Review by kekey1 — APPROVED** [review]

**Pull request merged**